python/ray/tune/tests/test_actor_reuse.py

```diff
@@ -134,15 +101,6 @@ def testTrialReuseEnabled(self):
         self.assertEqual([t.last_result["iter"] for t in trials], [2, 2, 2, 2])
         self.assertEqual([t.last_result["num_resets"] for t in trials], [4, 5, 6, 7])
 
-    def testTrialReuseEnabledFunction(self):
-        num_resets = defaultdict(lambda: 0)
-        trials = self._run_trials_with_frequent_pauses(
-            create_resettable_function(num_resets), reuse=True
-        )
-        self.assertEqual([t.last_result["id"] for t in trials], [0, 1, 2, 3])
-        self.assertEqual([t.last_result["iter"] for t in trials], [2, 2, 2, 2])
-        self.assertEqual([num_resets[t.trial_id] for t in trials], [0, 0, 0, 0])
```

Comment:
As you can see, this test doesn't work - the number of resets should have been 4, 5, 6, 7